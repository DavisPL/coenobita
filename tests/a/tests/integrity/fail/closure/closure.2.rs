#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::integrity({*}{*} fn({root}{root}) -> {root}{root})]
    let f = |x: i32| x + 1;

    #[cnbt::integrity({b}{b})]
    let y = f(5);
}