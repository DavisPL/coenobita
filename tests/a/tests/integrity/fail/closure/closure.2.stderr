error: expected {b}{b}, found {bin}{bin}
 --> tests/integrity/fail/closure/closure.2.rs:9:13
  |
9 |     let y = f(5);
  |             ^^^^
